app/src/main/java/com/amaze/filemanager/filesystem/HybridFile.java

```diff
@@ -470,11 +470,9 @@ public Boolean execute(SFTPClient client) throws IOException {
                       .stat(SshClientUtils.extractRemotePathFrom(path))
                       .getType()
                       .equals(FileMode.Type.DIRECTORY);
-                } catch (SFTPException notFound) {
```

Comment:
What happens in case of `SFTPException` why did you remove it?

Reply:
`net.schmizz.sshj.sftp.SFTPException` extends `net.schmizz.sshj.common.SSHException`, which extends `java.io.IOException`.